include/Options/Business/Force_Lang.php

```diff
@@ -6,15 +6,38 @@
 namespace WP_Syntex\Polylang\Options\Business;
 
 use WP_Syntex\Polylang\Options\Abstract_Option;
+use WP_Syntex\Polylang\Options\Options;
 
 defined( 'ABSPATH' ) || exit;
 
 /**
  * Class defining the "Determine how the current language is defined" option.
+ * /!\ Constructor depends on `hide_language_from_content_option`: this option must be set AFTER `hide_language_from_content_option`.
  *
  * @since 3.7
  */
 class Force_Lang extends Abstract_Option {
+	/**
+	 * @var array
+	 */
+	private $enum = array( 0, 1, 2, 3 );
```

Comment:
i'd do the opposite and use `array( 1, 2, 3 )` as default.
Since it'll be the default behavior of Polylang in 3.7.

Reply:
Done in [beb70ec](https://github.com/polylang/polylang/pull/1431/commits/beb70ecf2d0c380bf2028bbb8210e97f59903ce5).